torch/utils/tensorboard/_pytorch_graph.py

```diff
@@ -240,45 +237,18 @@ def graph(model, args, verbose=False, operator_export_type='ONNX', omit_useless_
       args (tuple): input tensor[s] for the model.
       verbose (bool): Whether to print out verbose information while
         processing.
-      operator_export_type (str): One of 'ONNX', 'ONNX_ATEN', or 'RAW'.
-        Defaults to 'ONNX' format  because it outputs the most visually
-        understandable format.
-      omit_useless_nodes (boolean): Whether to remove nodes from the graph.
     """
-    operator_export_type = getattr(OperatorExportTypes, operator_export_type)
 
 
-    with torch.onnx.set_training(model, False):
+    with torch.onnx.set_training(model, False):  # TODO: move outside of torch.onnx?
```

Comment:
@suo FYI on this one.

Reply:
This could be moved out, there's no onnx–specific code in this contextmanager. Although I'm not sure how generally useful it is

Reply:
Yup, I am wondering will it break people's existing code after moving.

Reply:
We won't change it as a part of this PR. Potentially a good followup fix.